precommit/src/main/shell/plugins.d/hadolint.sh

```diff
@@ -95,8 +91,8 @@ function hadolint_logic
     args=(--no-color)
   fi
 
-  for i in "${HADOLINT_CHECKFILES[@]}"; do
-    if [[ -f "${i}" ]]; then
+  for i in "${CHANGED_FILES[@]}"; do
+    if [[ "${i}" =~ Dockerfile ]] && [[ -f "${i}" ]]; then
```

Comment:
Why scope it to file names like `Dockerfile`? The previous values contained in `HADOLINT_CHECKFILES` used to be pre-filtered according to this expression?

Reply:
Yes.  HADOLINT_CHECKFILES was basically everything that had Dockerfile in the name.  There is a slight performance penalty not prefiltering... but only if there are no other files being excluded.  As soon as exclusion is introduced, a second loop is required and that quickly makes the perf diff a negative since the exclusion list may actually be bigger than what is in the patch!